tvb/simulator/integrators.py

```diff
@@ -88,6 +88,17 @@ class Integrator(core.Type):
         it is consitent with Monitors using sample periods corresponding to
         powers of 2 from 128 to 4096Hz.""")
 
+    constraint_state_variable_indices = arrays.IntegerArray(
```

Comment:
constrained or bounded would be better choice than constraint

Reply:
Done!